tests.js

```diff
@@ -46,16 +46,16 @@ describe('Order Book', () => {
       expect(updatedBook).to.deep.equal([{ type: 'sell', quantity: 12, price: 5950 }])
     })
 
-    it('fulfills an order and reduces the matching order when the book contains a matching order of a larger quantity', () => {
+    it('5. fulfills an order and reduces the matching order when the book contains a matching order of a larger quantity', () => {
       const existingBook = [{ type: 'buy', quantity: 15, price: 6150 }, { type: 'sell', quantity: 12, price: 6950 }]
       const incomingOrder = { type: 'sell', quantity: 10, price: 6150 }
 
       const updatedBook = reconcileOrder(existingBook, incomingOrder)
 
-      expect(updatedBook).to.deep.equal([{ type: 'sell', quantity: 12, price: 6950 }, { type: 'buy', quantity: 5, price: 6150 }])
+      expect(updatedBook).to.deep.equal([{ type: 'buy', quantity: 5, price: 6150 }, { type: 'sell', quantity: 12, price: 6950 }])
```

Comment:
Changing the order shouldn't be necessary, I told some folks who were struggling that it was fine but just know that you shouldn't have to.